Source/Core/Core/PowerPC/JitCommon/JitBase.h

```diff
@@ -39,6 +41,48 @@
 #define JITDISABLE(setting)                                                                        \
   FALLBACK_IF(SConfig::GetInstance().bJITOff || SConfig::GetInstance().setting)
 
+struct RangeTracker
+{
+  rangeset<size_t> ranges;
+
+  void add(const u8* begin, const u8* end) { ranges.add((size_t)(begin), (size_t)(end)); }
```

Comment:
These function names don't obey our codebase coding style.

Also, C-style casts should be avoided and replaced with static_cast or a functional style cast.